### ISSUE TITLE
the embeddings of the same sentence are different

### ISSUE THREAD

**Issue by jkl375** [thunlp/LegalPLMs#1]
I try run the code by python, but the embeddings of the same sentence are different.
>>> from transformers import AutoModel, AutoTokenizer
>>> tokenizer = AutoTokenizer.from_pretrained("hfl/chinese-roberta-wwm-ext")
>>> model = AutoModel.from_pretrained("thunlp/Lawformer")
>>> inputs = tokenizer("任某提起诉讼，请求判令解除婚姻关系并对夫妻共同财产进行分割。", return_tensors="pt")
>>> outputs = model(**inputs)

**Comment by xcjthu** [comment]
The embeddings may be affected by dropout. Could you run `model=model.eval()` before inference?
